ci-operator/step-registry/ipi/libvirt/pre/ipi-libvirt-pre-chain.yaml

```diff
@@ -2,10 +2,23 @@ chain:
   as: ipi-libvirt-pre
   steps:
   - ref: ipi-conf-libvirt
+  - ref: ipi-conf-debug-kdump-configure-logs
   - chain: ipi-install-libvirt
   env:
   - name: ARCH
   - name: BRANCH
+  - name: CRASH_KERNEL_MEMORY
```

Comment:
is this only impacting multi-arch libvirt or x86 as well?

Reply:
Only us - this is the full list of affected tests:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pj-rehearse/openshift/release/42882/a46435fc0206afc1787b84161b370251afe3b567

The ipi-libvirt-pre chain is only used by us.

Reply:
This is the workflow used by the CRC guys:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/code-ready/crc/e2e/code-ready-crc-e2e-workflow.yaml